install/installer/pkg/config/v1/config.go

```diff
@@ -99,6 +100,8 @@ type Config struct {
 
 type Metadata struct {
 	Region string `json:"region" validate:"required"`
+	// InstallationShortname establishes the "identity" of the (application) cluster.
+	InstallationShortname *string `json:"shortname,omitempty"`
```

Comment:
```suggestion
	InstallationShortname *string `json:"shortname,omitempty" validate:"required"`
```

If this is a required field, please add this. The `omitempty` can probably be removed and, strictly speaking, it doesn't need to be a pointer (apologies for that, I misunderstood the requirement).

If you'd prefer me to make the change, happy to do so

Reply:
We sorted out in sync:
 - `defaults()` is currently only used to initialize the very first config file
 - but _not_ to merge it as base "under" an already existing one
 - 
So we mark this as "required" to be in line with current semantics and usage. :heavy_check_mark: